src/platforms/hosted/cli.c

```diff
@@ -133,76 +134,102 @@ static void cl_help(char **argv)
 	PRINT_INFO(
 		"\n"
 		"Usage: %s [-h | -l | [-vBITMASK] [-d PATH | -P NUMBER | -s SERIAL | -c TYPE]\n"
-		"  [-n NUMBER] [-j] [-C] [-t | -T] [-e] [-p] [-R[h]] [-H] [-M STRING ...]\n"
-		"  [-E | -w | -V | -r] [-a ADDR] [-S number] [file]]\n"
+		"\t[-n NUMBER] [-j] [-C] [-t | -T] [-e] [-p] [-R[h]] [-H] [-M STRING ...]\n"
+		"\t[-f | -m] [-E | -w | -V | -r] [-a ADDR] [-S number] [file]]\n"
 		"\n"
-		"Single-shot and verbosity options [-h | -l | -vBITMASK]:\n"
-		"\t-h              Show the version version and this help, then exit\n"
-		"\t-l              List available supported probes\n"
-		"\t-v<bitmask>     Set the output verbosity level based on some combination of:\n"
-		"\t                  1 = INFO, 2 = GDB, 4 = TARGET, 8 = PROBE, 16 = WIRE\n"
+		"\t-h, --help       Show the version and this help, then exit\n"
+		"\t-l, --list       List available supported probes\n"
+		"\t-v, --verbose    <bitmask> Set the output verbosity level based on some combination of:\n"
```

Comment:
This looks good, but think we'd like to suggest either dropping the `<bitmask>` part (as it's shown in the short help for the section) or show it as `-v<bitmask>` so people understand where it goes for the shortopt